hbase-server/src/main/java/org/apache/hadoop/hbase/wal/AbstractWALRoller.java

```diff
@@ -223,8 +223,7 @@ public void run() {
               if (waitingTime < rollWaitTimeout && nAttempts < maxRollRetry) {
                 nAttempts++;
                 LOG.warn("Retry to roll log, nAttempts={}, waiting time={}ms, sleeping 1s to retry,"
-                    + " last excepiton= {}", nAttempts, waitingTime,
-                  ioe.getCause().getClass().getSimpleName());
+                    + " last excepiton= {}", nAttempts, waitingTime, ioe);
```

Comment:
nit: no need to add `{}` after `last excepiton`, keeping `ioe` as the last arg will print the whole Exception stacktrace.

Reply:
Thanks, I'll remove it.